README.md

```diff
@@ -63,7 +63,7 @@ Say your Electron app lives in `path/to/app`, and has a structure like this:
 You now run `electron-packager` to build the app for Red Hat:
 
 ```
-$ electron-packager . app --platform linux --arch x64 --out dist/
+$ electron-packager . app --platform linux --arch x86_64 --out dist/
```

Comment:
`electron-packager` is an external project, and it doesn't take `x86_64` for its `arch` flag. Check it out here: https://github.com/electron-userland/electron-packager/blob/master/usage.txt#L20